apps/nn_ops/DepthwiseConvolution.cpp

```diff
@@ -0,0 +1,191 @@
+#include <assert.h>
+#include <malloc.h>
+#include <memory.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <cmath>
+#include <limits>
+
+#include "halide_benchmark.h"
+
+#include "DepthwiseConvolution.h"
+
+#include "HalideBuffer.h"
+
+int32_t saturating_rounding_doubling_high_multiply(int32_t a, int32_t b) {
```

Comment:
Is it possible to share the same saturating_rounding_doubling_high_multiply(), rounding_shift_right(), and  multiply_quantized_multiplier() between some of the .cpp?

Reply:
Yes, good point! Thank you for suggestion.